gradle/spotbugs-exclude.xml

```diff
@@ -547,5 +547,10 @@ For a detailed description of spotbugs bug categories, see https://spotbugs.read
         </Or>
         <Bug pattern="RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT"/>
     </Match>
+    <Match>
+        <!-- Suppress warning about using static DateFormat in a multithreaded context -->
+        <Class name="org.apache.kafka.tools.ReplicaVerificationTool$ReplicaBuffer"/>
+        <Bug pattern="STCAL_INVOKE_ON_STATIC_DATE_FORMAT_INSTANCE"/>
+    </Match>
```

Comment:
Is this safe? There is a reason for that warning. Why should we suppress it?

Reply:
No, it might be a concern, but I suppress it since this test is going to be deleted. 
I don't think it's very meaningful to fix it at this moment. 🤔 

https://github.com/apache/kafka/blob/bb6ebd83f9a0f9b2a37704f1f37b78a48cf480e1/tools/src/main/java/org/apache/kafka/tools/ReplicaVerificationTool.java#L115